### PR TITLE
Add compat data for TextMetrics

### PR DESCRIPTION
**Pull request by maboa**
Note:

I added ```actualBoundingBoxLeft,
actualBoundingBoxRight,
fontBoundingBoxAscent,
fontBoundingBoxDescent,
actualBoundingBoxAscent,
actualBoundingBoxDescent,
emHeightAscent,
emHeightDescent,
hangingBaseline,
alphabeticBaseline,
ideographicBaseline``` as one field reflecting the table here : https://developer.mozilla.org/en-US/docs/Web/API/TextMetrics

But happy to separate into individual entries.

### CODE REVIEW

File: api/TextMetrics.json [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "actualBoundingBoxLeft, actualBoundingBoxRight, fontBoundingBoxAscent, fontBoundingBoxDescent, actualBoundingBoxAscent, actualBoundingBoxDescent, emHeightAscent, emHeightDescent, hangingBaseline, alphabeticBaseline, ideographicBaseline": {
```
Comment (Elchi3) [inline api/TextMetrics.json]: This is not a good identifier. Please rename this to `advanced_text_metrics` and add a description: 
`"description": "Advanced text metrics properties",`

File: api/TextMetrics.json [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+        "__compat": {
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": true,
+              "notes": "To turn on advanced text metrics, set the flag <code>ExperimentalCanvasFeatures</code> to <code>true</code>."
```
Comment (Elchi3) [inline api/TextMetrics.json]: This needs to be a `flags` object instead of a note.

File: api/TextMetrics.json (hunk 1 above)
Comment (maboa) [reply api/TextMetrics.json]: 👍

File: api/TextMetrics.json (hunk 2 above)
Comment (maboa) [reply api/TextMetrics.json]: 👍

**Comment by maboa** [comment]
Done :)

**Pull request merged**